src/reducers/systemIntakeReducer.ts

```diff
@@ -38,6 +39,23 @@ function systemIntakeReducer(
         ...state,
         isLoading: false
       };
+    case removeSystemIntake.TRIGGER:
+      return {
+        ...state,
+        systemIntake: initialSystemIntakeForm,
+        isLoading: false,
+        error: null
+      };
+    case removeSystemIntake.FAILURE:
```

Comment:
We don't need the `removeSystemIntake.FAILURE` nor the `removeSystemIntake.FULFILL` cases here because they are never invoked.

In the EASi app, `ROUTINE.failure()` and `ROUTINE.failure.fulfill()` are generally called when an API request is made. `.falure()` for when the request fails and `.fulfill()` after the success is complete. (See try catch finally in sagas files)

Since removing the system intake from redux doesn't make an API request, the `.success()`, `.failure()`, and `.fulfill()` are never called.